app/models/organization.rb

```diff
@@ -16,6 +16,7 @@ class Organization < ApplicationModel
   include Organization::SearchIndex
 
   has_many                :members,  class_name: 'User'
+  has_and_belongs_to_many :members,   after_add: :cache_update, after_remove: :cache_update, class_name: 'User'
```

Comment:
There are two associations with the same name `members` which looks wrong to me.